calculator.js

```diff
@@ -0,0 +1,60 @@
+(function() {
+
+var writes = document.getElementById('screen');
```

Comment:
In programming a good practice is to separate program into smaller and reusable modules - classes. In that way it is easier to develop, maintain and improve the code base. We are going to use *screen* as an example for that. Your task is to create a **class** *Screen* which will encapsulate all functionality of a screen. Once you define it we are going to use it to interact with the screen.

Here is an example how it should work:
```
var screen = new Screen();
...
screen.setNumber(12);
```
And than in the future we can add new functionalities, for example if there is an error we could say:
```
screen.setErrorColor();
```

Couple of hints:
1) Check how **prototypes** work in JS
2) Screen should use element 'screen' under the hood